composer.json

```diff
@@ -10,10 +10,10 @@
     ],
     "require": {
         "guzzle/http": "~3.7",
-        "jms/serializer": "0.12.*",
+        "jms/serializer": "~1.1",
         "psr/log": "~1.0",
-        "symfony/console": "~2.3",
-        "symfony/expression-language": "~2.4"
+        "symfony/console": "~2.8",
+        "symfony/expression-language": "~2.8"
```

Comment:
@lyrixx it might be worth adding support for Symfony 3 components here too.

Btw, when building the phar, be careful about using deps compatible with your target PHP version, not with your local one (the composer.json currently does not have this setting)